deployments/polygon/usdc/deploy.ts

```diff
@@ -0,0 +1,74 @@
+import { Deployed, DeploymentManager } from '../../../plugins/deployment_manager';
+import { DeploySpec, deployComet } from '../../../src/deploy';
+
+const secondsPerDay = 24 * 60 * 60;
+
+const MAINNET_TIMELOCK = '0x6d903f6003cca6255d85cca4d3b5e5146dc33925';
+
+export default async function deploy(deploymentManager: DeploymentManager, deploySpec: DeploySpec): Promise<Deployed> {
+  const trace = deploymentManager.tracer()
+  const ethers = deploymentManager.hre.ethers;
+
+  // pull in existing assets
+  const USDC = await deploymentManager.existing('USDC', '0x2791bca1f2de4661ed88a30c99a7a9449aa84174', 'polygon');
+  const WETH = await deploymentManager.existing('WETH', '0x7ceb23fd6bc0add59e62ac25578270cff1b9f619', 'polygon');
+  const WBTC = await deploymentManager.existing('WBTC', '0x1bfd67037b42cf73acf2047067bd4f2c47d9bfd6', 'polygon');
+  const DAI = await deploymentManager.existing('DAI', '0x8f3cf7ad23cd3cadbd9735aff958023239c6a063', 'polygon');
+  const USDT = await deploymentManager.existing('USDT', '0xc2132d05d31c914a87c6611c10748aeb04b58e8f', 'polygon');
```

Comment:
We're getting rid of these right, but is that for a different branch?

Reply:
yep, different branch